ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.ts

```diff
@@ -355,6 +354,16 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'DSR Skyblind',
+      // 631A Skyblind (2.2s cast) is a targetted ground aoe where A65 Skyblind
+      // effect expired on the player.
+      type: 'GainsEffect',
+      netRegex: NetRegexes.gainsEffect({ effectId: 'A65' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: (_data, matches) => parseFloat(matches.duration) - 0.2,
```

Comment:
For similar reasons to https://github.com/quisquous/cactbot/pull/4408#issuecomment-1120243311, in general I don't want to be in the business of trying to account for reaction times.  I'd rather have the call happen when you're supposed to do the thing, even if that adds a little latency into the process.

You've got a lot of time to move out of the baited circle here too (vs the baited protean above), so I think no need for the reaction time adjustment here.